website/src/css/custom.css

```diff
@@ -29,6 +30,155 @@ header>h1 {
     font-size: 1.2rem !important;
 }
 
+.panel {
+    border-top: 2px solid #ddd;
+    padding: 20px 20px 25px 20px;
+}
+
+.panel>a,
+.panel>h2 {
+    text-transform: uppercase;
+    font-size: 20px;
+    letter-spacing: 0.7px;
+    font-weight: 500;
+    margin-bottom: 0;
+    line-height: 33px;
+}
+
+.panel>a,
+.panel>h2 {
+    display: inline-block;
+}
+
+.panel>h3 {
+    display: block;
+    text-transform: uppercase;
+    font-size: 17px;
+    letter-spacing: 0.6px;
+    font-weight: 500;
+    margin-bottom: 0;
+    line-height: 28px;
+}
+
+.panel>p {
+    font-size: 15px;
+    margin-bottom: 0;
+}
+
+.panel.section-title {
+    background-color: #f9f9f9;
+    border-left: 2px solid #ddd;
+    border-right: 2px solid #ddd;
+}
+
+.panel.secondary-panel.section-title {
+    background-color: #fefefe;
+    border-left: 2px solid #ddd;
+    border-right: 2px solid #ddd;
+}
+
+.panel.secondary-panel {
+    padding: 15px 20px 20px 20px;
+}
+
+.content-status {
+    display: block;
+    width: 425px;
+    max-width: 100%;
+    padding: 10px;
+    border: 1px solid #ddd;
+    background-color: #fcfcfc;
+    font-size: 14px;
+    border-radius: 5px;
+    margin-top: 50px;
+    margin-left: auto;
+}
+
+.update-request {
+    margin-left: auto;
+    display: block;
+    text-align: right;
+    margin-top: 10px;
+}
+
+.content-status>p {
+    margin-bottom: 0;
+}
+
+.content-status .status-meta {
+    display: inline-block;
+    width: 130px;
+}
+
+.lastUpdated_3DPF {
+    display: none;
+}
+
+nav.pagination-nav {
+    margin-top: 0;
+    margin-bottom: 20px;
+}
+
+.markdown {
+    margin-bottom: 30px !important;
+}
+
+footer {
+    padding: 10px !important;
+    display: none !important;
+    /* can remove display style if we want to reintroduce */
+}
+
+footer>.col {
+    text-align: right;
+    padding-right: 6px !important;
+}
+
+footer>.col a,
+.update-request>a {
+    padding: 10px;
+    border: 1px solid #ddd;
+    background-color: #fcfcfc;
+    font-size: 14px;
+    border-radius: 5px;
+    margin-top: 10px;
+    text-decoration: none;
+    transition-property: all;
+    transition-duration: 0.2s;
+    display: inline-block;
+}
+
+.update-request>a {
+    margin-bottom: 10px;
+}
+
+footer>.col a:hover,
+.update-request>a:hover {
+    text-decoration: none;
+    background-color: #f8f8f8;
+    border: 1px solid #3578e5;
+}
+
+footer>.col a svg {
+    height: 17px;
+    margin-bottom: 1px;
+}
+
+
+footer.docusaurus-mt-lg {
+    margin-top: 0 !important;
+}
+
+.footer__logo {
```

Comment:
I decided to keep these styles in case we ever decide to reintroduce the footer (eg with socials or other bits related to our branding update). Can remove if preferred, we can always pull from history.